app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
This PR does multiple things: set up user management and do posts management. That's too many things in a PR, you should strive for a better PR management and make sure PRs are focused on a single subject. Remove unnecessary views and mailers